Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM alpine:latest as build
 RUN apk --update add ca-certificates
 
 COPY sopsinstall.sh /tmp/sopsinstall.sh
-RUN /tmp/sopsinstall.sh -b /usr/local/bin
+RUN sh /tmp/sopsinstall.sh -b /usr/local/bin
```

Comment:
for some reason locally the sh file isn't executable with the latest goreleaser (might need to put a PR in for them). we know its a shell so we can just use this.